### PR TITLE
1.0.0 doc changes

### PR DESCRIPTION
**Pull request by cs-abhishek-shukla**
- Rewrote usage, readme, and setup
        - Added navigation links

### CODE REVIEW

File: docs/setup.md [hunk 1]
```diff
@@ -5,16 +5,16 @@
 
 1. To install a widget, click **Content Hub** > **Discover**.
 
-2. From the list of solution pack that appears, search for and select **Global Visibility**. 
+2. From the list of widgets that appears, search for and select **Global Visibility**. 
 
-3. Click the card of the **Global Visibility** solution pack card.
+3. Click the **Global Visibility** widget card to begin installation.
```
Comment (cs-yash-patodkar) [inline docs/setup.md:10]: No use of this widget installed solely, It is meant for global visibility solution pack. And will be installed along with SP. We can remove the Installation Tag above

Same for the 2nd point too

**Review by cs-yash-patodkar — APPROVED** [review]

File: docs/setup.md [hunk 2]
```diff
@@ -5,16 +5,16 @@
 
 1. To install a widget, click **Content Hub** > **Discover**.
 
-2. From the list of solution pack that appears, search for and select **Global Visibility**. 
+2. From the list of widgets that appears, search for and select **Global Visibility**.
```
Comment (cs-yash-patodkar) [inline docs/setup.md:8]: No need search for the widget and install, it comes with the SP only

File: docs/setup.md (hunk 1 above)
Comment (cs-abhishek-shukla) [reply docs/setup.md:10]: Even if the widget is part of an SP, we still document its installation process. For consistency and if in future, it evolves to be something independent.

File: docs/setup.md (hunk 2 above)
Comment (cs-abhishek-shukla) [reply docs/setup.md:8]: Even if the widget is part of an SP, we still document its installation process. For consistency and if in future, it evolves to be something independent.

Comment (cs-yash-patodkar) [reply docs/setup.md:8]: Okay

**Review by cs-yash-patodkar — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -10,8 +10,6 @@
 
 - **Applicable**: Dashboards
```
Comment (cs-yash-patodkar) [inline README.md]: Can you remove this dashboard line, not applicable on dashboard

Comment (cs-abhishek-shukla) [reply README.md]: ok. Fixed.

**Review by cs-yash-patodkar — APPROVED** [review]

**Review by cs-suyog-jinturkar — APPROVED** [review]
Already Approved

**Pull request merged**